module/PowerNSX.psm1

```diff
@@ -9111,13 +9111,15 @@ function Get-NsxLogicalSwitch {
     Transport Zone.
 
     .EXAMPLE
-
-    Example1: Get a named Logical Switch
     PS C:\> Get-NsxTransportZone | Get-NsxLogicalswitch -name LS1
```

Comment:
Need to remove PS C:\> string from first list following .EXAMPLE get-command adds this automatically.